src/s11/handlers/create_bearer_resp_handler.c

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) 2020, Infosys Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <arpa/inet.h>
+#include <unistd.h>
+#include <string.h>
+#include <pthread.h>
+
+#include "log.h"
+#include "err_codes.h"
+#include "ipc_api.h"
+#include "gtpv2c.h"
+#include "gtpv2c_ie.h"
+#include "s11_config.h"
+#include "msgType.h"
+#include "s11_options.h"
+#include <gtpV2StackWrappers.h>
+#include "gtp_cpp_wrapper.h"
+
+/****Globals and externs ***/
+
+/*S11 CP communication parameters*/
+extern int g_s11_fd;
+extern struct sockaddr_in g_s11_cp_addr;
+extern s11_config_t g_s11_cfg;
+extern socklen_t g_s11_serv_size;
+extern volatile uint32_t g_s11_sequence;
+
+struct thread_pool *g_tpool;
+
+extern struct GtpV2Stack* gtpStack_gp;
+extern volatile uint32_t g_s11_sequence;
+
+/****Global and externs end***/
+
+/**
+* Stage specific message processing.
+*/
+static int
+cb_resp_processing(struct CB_RESP_Q_msg *cb_resp_msg)
+{
+	
+	struct MsgBuffer*  cbRespMsgBuf_p = createMsgBuffer(S11_MSGBUF_SIZE);
+	if(cbRespMsgBuf_p == NULL)
+	{
+        log_msg(LOG_ERROR, "Error in initializing msg buffers required by gtp codec.\n");
+        return -1;
+    }
+	
+	GtpV2MessageHeader gtpHeader;
+	gtpHeader.msgType =  GTP_CREATE_BEARER_RSP;
+	gtpHeader.sequenceNumber = g_s11_sequence;
```

Comment:
May be we should use the sequence number received in CB Req?

Reply:
Yes, corrected.